waspc/data/Generator/templates/react-app/package.json

```diff
@@ -7,9 +7,8 @@
   {=& depsChunk =},
   {=& devDepsChunk =},
   "scripts": {
-    "start": "npm run validate-env && vite",
```

Comment:
Removing the `validate-env.mjs` script in favor of doing it inside of a Vite plugin: https://github.com/wasp-lang/wasp/pull/2392